source/FAQ/index.rst

```diff
@@ -68,13 +86,24 @@ https://clearlinux.org/news-blogs/where-etcfstab-clear-linux
 
 |
 
-Software packages
-*****************
+Does it use the Intel Compiler (icc)?
+=====================================
+
+No. |CL| uses open source compilers: :command:`gcc` and :command:`clang`. |CL|
+does not compile any packages with :command:`icc`. 
+
+For a more detailed explanation, see this discussion on the community forum:
+https://community.clearlinux.org/t/does-clear-linux-os-use-the-intel-compiler-icc-tl-nope/
```

Comment:
Embed hyperlink.  on Ln 95:
... see `this discussion`_ on the community forum. 
Add hyperlink at bottom of document: 
`.. _this discussion: https://community.clearlinux.org/t/does-clear-linux-os-use-the-intel-compiler-icc-tl-nope/`